tests/app/questionnaire/test_date_rules.py

```diff
@@ -3,153 +3,149 @@
-        )
-        self.assertTrue(result)
-
-        answer_value = "2016-06-12"
-        result = evaluate_date_rule(
-            when, None, get_schema_mock(), metadata, answer_value
-        )
-        self.assertFalse(result)
-
-    def test_evaluate_date_rule_greater_than_with_id(self):
-        when = {
-            "id": "date-answer",
-            "condition": "greater than",
-            "date_comparison": {"id": "compare_date_answer"},
-        }
-
-        answer_store = AnswerStore({})
-        answer_store.add_or_update(
-            Answer(answer_id="compare_date_answer", value="2018-02-03")
-        )
-
-        answer_value = "2018-02-04"
-        result = evaluate_date_rule(
-            when, answer_store, get_schema_mock(), None, answer_value
-        )
-        self.assertTrue(result)
-
-        answer_value = "2018-02-03"
-        result = evaluate_date_rule(
-            when, answer_store, get_schema_mock(), None, answer_value
-        )
-        self.assertFalse(result)
-
-    def test_evaluate_date_rule_invalid(self):
-        when = {
-            "id": "date-answer",
-            "condition": "greater than",
-            "date_comparison": {"id": "compare_date_answer"},
-        }
-        answer_store = AnswerStore({})
-        result = evaluate_date_rule(when, answer_store, get_schema_mock(), None, None)
-
-        self.assertFalse(result)
-
-    def test_do_not_go_to_next_question_for_date_answer(self):
-        goto_rule = {
-            "id": "next-question",
-            "when": [
-                {
-                    "id": "date-answer",
-                    "condition": "equals",
-                    "date_comparison": {"value": "2018-01"},
-                }
-            ],
-        }
-
-        answer_store = AnswerStore({})
-        answer_store.add_or_update(Answer(answer_id="date-answer", value="2018-02-01"))
-
-        current_location = Location(section_id="some-section", block_id="some-block")
-
-        self.assertFalse(
-            evaluate_goto(
-                goto_rule=goto_rule,
-                schema=get_schema_mock(),
-                metadata={},
-                answer_store=answer_store,
-                list_store=ListStore(),
-                current_location=current_location,
-            )
-        )
+
+
+def test_evaluate_date_rule_equals_with_value_now(questionnaire_schema):
+    when = {
+        "id": "date-answer",
+        "condition": "equals",
+        "date_comparison": {"value": "now"},
+    }
+
+    answer_value = datetime.now(tz=timezone.utc).strftime("%Y-%m-%d")
+    result = evaluate_date_rule(when, None, questionnaire_schema, None, answer_value)
+    assert result
+
+    answer_value = "2000-01-01"
+    result = evaluate_date_rule(when, None, questionnaire_schema, None, answer_value)
+    assert not result
+
+
+def test_evaluate_date_rule_equals_with_with_offset(questionnaire_schema):
+    when = {
+        "id": "date-answer",
```

Comment:
Similar but not the same, only that item in multiple dicts. If I make it a fixture then it will fragment the code so much as to be unreadable, I specifically avoided fixturing these objects because of this